src/api/cloudDb/enterprise/backup/backup.iceberg.service.js

```diff
@@ -0,0 +1,12 @@
+angular.module('ovh-api-services').service('OvhApiCloudDBEnterpriseBackupIceberg', (iceberg) => {
+  const backupResource = iceberg('/cloudDB/enterprise/cluster/:clusterId/backup/:backupId', {
+    clusterId: '@clusterId',
+    backupId: '@backupId',
+  }, {
+    post: { method: 'POST' },
```

Comment:
You definitely need an interceptor here 😄

Reply:
Since these are Iceberg calls, I guess iceberg handles the caching. Please correct me if I am wrong.